src/core/reactor.cc

```diff
@@ -1241,12 +1256,12 @@ cpu_stall_detector_linux_perf_event::start_sleep() {
 }
 
 bool
-cpu_stall_detector_linux_perf_event::reap_event_and_check_spuriousness() {
-    struct read_format {
-        uint64_t value;
-    } buf;
-    _fd.read(&buf, sizeof(read_format));
```

Comment:
We lost this read. I don't remember the details of perf_event, but won't it get constipated if we don't relieve it of buffered data?

Reply:
No, there is no actual stream of results or anything like that: the `_fd` just returns the current data on any read no matter the offset: reading is totally optional.

The read path is here:

https://elixir.bootlin.com/linux/v5.15.86/source/kernel/events/core.c#L5521

Reply:
Yes, the documentation agrees.

Reply:
Interesting, I couldn't even find the doc probably because the man page has grown pretty unwieldy.